neofetch

```diff
@@ -2721,6 +2722,22 @@ get_memory() {
     esac
 }
 
+get_bootloader() {
+    case $os in
+        "Linux")
+            if [[ -d "/boot/grub" ]]; then
```

Comment:
This will fail on openSUSE (and maybe others too), since there the directory is named `/boot/grub2`.